justfile

```diff
@@ -81,11 +71,11 @@ _tools-install tool cmd:
     sort {{ INSTALLED_TOOLS }} -o {{ INSTALLED_TOOLS }}
 
 # a the placeholder `{}` which is the path to the correct module.
-_invokeMod cmd target='all':
+_invokeMod cmd:
     #!/usr/bin/env bash
     set -euo pipefail # makes scripts act like justfiles (https://github.com/casey/just#safer-bash-shebang-recipes)
-    if [ "{{ target }}" = "{{ PKG_ROOT }}" ]; then
-      xargs -L1 -P 8 -t -I {} {{ cmd }} <<< "{{ MODS }}"
-     else
-      xargs -L1 -P 8 -t -I {} {{ cmd }} <<< "{{ target }}"
+    if [ "{{ CURRENT_DIR }}" = "{{ PKG_ROOT }}" ]; then
+      go list -f '{{{{.Dir}}}}' -m | xargs -L1 -P 8 -t -I {} {{ cmd }}
```

Comment:
```suggestion
      go list -f '{{{{.Dir}}' -m | xargs -L1 -P 8 -t -I {} {{ cmd }}
```

Reply:
justfile uses asymmetric escaping for curly braces 

https://github.com/casey/just#escaping-